website/src/components/SearchPage/DownloadDialog/DownloadForm.tsx

```diff
@@ -269,3 +280,32 @@ export const DownloadForm: FC<DownloadFormProps> = ({
         </div>
     );
 };
+
+function getSequenceNames(
+    referenceGenomesSequenceNames: ReferenceGenomesSequenceNames,
+    selectedSuborganism: string | null,
+): { nucleotideSequences: SequenceName[]; genes: SequenceName[]; useMultiSegmentEndpoint: boolean } {
+    if (SINGLE_REFERENCE in referenceGenomesSequenceNames) {
+        const { nucleotideSequences, genes } = referenceGenomesSequenceNames[SINGLE_REFERENCE];
+        return {
+            nucleotideSequences: nucleotideSequences.map(getSinglePathogenSequenceName),
+            genes: genes.map(getSinglePathogenSequenceName),
+            useMultiSegmentEndpoint: isMultiSegmented(nucleotideSequences),
+        };
+    }
+
+    if (selectedSuborganism === null) {
+        return {
+            nucleotideSequences: [],
+            genes: [],
+            useMultiSegmentEndpoint: false, // LAPIS is multisegmented, since we're in multi pathogen case here. Use the "download all segments" endpoint which we get by pretending we're single segmented.
```

Comment:
```suggestion
            useMultiSegmentEndpoint: false, // LAPIS is multisegmented, since we're in multi pathogen case here, but we want to use the "download all segments" endpoint which we get by pretending we're single segmented.
```

I think linking the two sentences with a "but" helps my brain make sense of it, but nbd

Reply:
I already changed the comment to something else